### PR TITLE
Updated UI 'https://github.com/digi-serve/plugin_ABDesigner/issues/57'

### PR DESCRIPTION
**Pull request by Tacchinardi20**

### CODE REVIEW

**Comment by Achoobert** [comment]
So I added the "by field" option because the select manager is being used in the process tasks.

I think for exporting /managing module access it should be hidden as an option??
Perhaps only show it if the manager receives a user field list?

File: src/rootPages/Designer/ui_choose_form.js [hunk 1]
```diff
@@ -42,422 +75,487 @@ export default function (AB, init_settings) {
...
+               },
             },
          };
       } // ui()
 
       init(AB) {
          this.AB = AB;
 
-         this.$form = $$(this.ids.form);
+         this.$win = this.AB.Webix.ui(this.ui());
```
Comment (nh758) [inline src/rootPages/Designer/ui_choose_form.js:491]: I don't think we need this line. I don't see `this.$win` used anywhere

**Review by nh758 — CHANGES_REQUESTED** [review]
Overall looking good. Some things to fix.

**Comment by Tacchinardi20** [comment]
Thanks, Nathan

**Pull request merged**